app/components/DbEntry/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+*
+* DbEntry
+*
+*/
+
+import React from 'react';
+import styled from 'styled-components';
+import { getActive } from 'themes';
+import PropTypes from 'prop-types';
+
+class DbEntry extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {
+    super(props);
+    this.minmax = this.minmax.bind(this);
+    this.state = { toggle: false };
+  }
+  minmax(evt) {
```

Comment:
Not sure which naming convention we are using for function names but we should decide on one. Also the fill function seems to be useless in it's current state.